### PR TITLE
Modifying the limit size of yaml and GRPC inbound message

### PR DESCRIPTION
**Pull request by ooi22**
### Fix <When the size of configuration files exceeds 4M, the OAP get configuration fail>
- [ ] Add a unit test to verify that the fix works.
- [x] The default size is 4M,modifying the limit size to maximum.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Which configuration file is over 4M? Alerting rule?

**Comment by wu-sheng** [comment]
This change should be applied for certain use case, rather than making all YAML files adopting this change.
And the max length should not be hard coded, it should be configurable. 

For example, if you are talking about alerting case, we should make alerting file reader to accept a configurable size as a parameter.

Your current change is too reckless, and could cause OOM easily for loading a wrong file or malicious file

**Comment by Superskyyy** [comment]
> Your current change is too reckless, and could cause OOM easily for loading a wrong file or malicious file

This is a very valid point, please adjust to a setting instead of hardcoded. @ooi22

**Comment by ooi22** [comment]
> Which configuration file is over 4M? Alerting rule?

yes

**Comment by wu-sheng** [comment]
> > Which configuration file is over 4M? Alerting rule?
> 
> yes

Please follow the comment above.

**Comment by ooi22** [comment]
done

> > Your current change is too reckless, and could cause OOM easily for loading a wrong file or malicious file
> 
> This is a very valid point, please adjust to a setting instead of hardcoded. @ooi22

Done.

**Comment by wu-sheng** [comment]
You removed YAML resolver relative changes? Are those irrelevant?

**Comment by wu-sheng** [comment]
`configuration-vocabulary.md` and `changes.md` should be updated accordingly.

**Comment by ooi22** [comment]
> You removed YAML resolver relative changes? Are those irrelevant?

Yes, and it is another optimization point, but I have no time to change it to a setting instead of hardcoded, so I removed it.

**Comment by ooi22** [comment]
> `configuration-vocabulary.md` and `changes.md` should be updated accordingly.

Done

File: docs/en/setup/backend/configuration-vocabulary.md [hunk 1]
```diff
@@ -281,6 +281,7 @@ The Configuration Vocabulary lists all available configurations provided by `app
...
+| -                       | -             | maxInboundMessageSize                                                                                                                                                    | The max inbound message size of gRPC.                                                                                                                                                                                                                                                                                                                                                                                                                                      | SW_CONFIG_GRPC_MAX_INBOUND_MESSAGE_SIZE               | 4194304                                                                                      |
```
Comment (wu-sheng) [inline docs/en/setup/backend/configuration-vocabulary.md:284]:
```suggestion
| -                       | -             | maxInboundMessageSize                                                                                                                                                    | The max inbound message size of gRPC.                                                                                                                                                                                                                                                                                                                                                                                                                                      | SW_DCS_MAX_INBOUND_MESSAGE_SIZE               | 4194304                                                                                      |
```

File: oap-server/server-starter/src/main/resources/application.yml [hunk 2]
```diff
@@ -485,6 +485,7 @@ configuration:
     port: ${SW_DCS_SERVER_PORT:80}
     clusterName: ${SW_DCS_CLUSTER_NAME:SkyWalking}
     period: ${SW_DCS_PERIOD:20}
+    maxInboundMessageSize: ${SW_CONFIG_GRPC_MAX_INBOUND_MESSAGE_SIZE:4194304}
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/application.yml:488]:
```suggestion
    maxInboundMessageSize: ${SW_DCS_MAX_INBOUND_MESSAGE_SIZE:4194304}
```

**Review by wu-sheng — APPROVED** [review]
@ooi22 I changed your system env variable name to align with other configurations.

LGTM. @wankai123 PTAL.

**Review by wankai123 — APPROVED** [review]
LGTM

**Pull request merged**